sql2.txt

```diff
@@ -49,4 +80,10 @@ DELETE FROM OrderDetails WHERE OrderID IN (SELECT OrderID FROM Orders WHERE Cust
 
 What are the other two statements?  Try them out, and then paste them below.
 
+DELETE FROM orders WHERE customerid = 76;
+DELETE FROM customers WHERE customerid = 76;
 
+I am pretty sure it allowed me to delete the customer with the last statement. No error showed up and 
+when I tried to use a select statement to find the customer, nothing showed up. The database did still
+contain the orders though. The other statements also did not produce an error when I deleted. Were they
+supposed to?
```

Comment:
I have figured out what the problem was in the mentor session on Monday. Because I was not using sqlcommand.rb, I did not receive the error messages. These Foreign Key constraints messages are turned off by default in the regular CLI that I was using according to one of my classmates. Do I still need to run it again? Or redo the assignment with the correct file? I did check that customer with id 76 was missing from the customers, orders, and orderdetails tables. Let me know. Thanks.

Reply:
No, you **DO NOT** need to redo the assignment-- your answers are correct.

Yes, that is true. In the sqlite CLI you have to turn on foreign key constraints at runtime (which is wild to me) and `sqlcommand.rb` does that here:

https://github.com/mhope21/sqlintro/blob/3bafd70837f2914df90ed0cc56d9edbf1315ea35/sqlcommand.rb#L6

To do it in the sqlite CLI, you can run the same:

```sql
PRAGMA foreign_keys = ON;
```